Framework/HistogramData/inc/MantidHistogramData/HistogramDx.h

```diff
@@ -50,7 +50,9 @@ class MANTID_HISTOGRAMDATA_DLL HistogramDx
   HistogramDx(HistogramDx &&) = default;
   HistogramDx &operator=(const HistogramDx &) & = default;
   HistogramDx &operator=(HistogramDx &&) & = default;
-
+  bool operator==(const HistogramDx &rhs) const {
```

Comment:
@SimonHeybrock Just wanted to check you're okay with this and there's no reason it wasn't here in the first place?

Reply:
I think I once had a reason for not adding it, but right now I cannot think of a justification.

However, it should be added in the base class `FixedLengthVector` (or as a free function, taken from `HistogramDataTestHelper.h`), avoiding the duplication in all the derived types. Also `HistogramDataTestHelper.h` should be removed since it will no longer be required.

Reply:
Were you looking at something more like what it is now? @SimonHeybrock and @martyngigg

Reply:
Yes, looks good now, I think.